lib/addStyles.js

```diff
@@ -28,9 +28,19 @@ var getElement = (function (fn) {
 
 	return function(selector) {
 		if (typeof memo[selector] === "undefined") {
-			memo[selector] = fn.call(this, selector);
+			var styleTarget = fn.call(this, selector);
+			// Special case to return head of iframe instead of iframe itself
+			try {
+				if (styleTarget instanceof window.HTMLIFrameElement) {
```

Comment:
Yep, `style-loader` should only support environments which expose/mimic a DOM Interface, the tests already use `JSDOM` which is the primary non-browser target `style-loader` supports atm. @tobias-zucali Could you try without the `try/catch` and `instanceof` check and see if it works ? Maybe add a comment with the current solution in case it heavily breaks other people's setups and we need a hotfix :). Otherwise good to go imho